### ISSUE TITLE
[v1.0.0b] - `poetry install` fails in `docker-compose build`

### ISSUE THREAD

**Issue by smkent** [mealie-recipes/mealie#1691]
### First Check

- [X] This is not a feature request
- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the Mealie documentation, with the integrated search.
- [X] I already read the docs and didn't find an answer.

### What is the issue you are experiencing?

In attempting to build the Docker containers from source, I'm running `docker-compose up` (implying `docker-compose build`) on the `mealie-next` branch. The build fails on the `Dockerfile`'s `poetry install` step with:

```
Step 12/56 : RUN poetry install -E pgsql --no-dev
 ---> Running in b6bf57028510
Creating virtualenv mealie in /opt/pysetup/.venv
Installing dependencies from lock file

Package operations: 77 installs, 0 updates, 1 removal

  • Removing setuptools (65.4.0)
  • Installing six (1.16.0)

  JSONDecodeError

  Expecting value: line 1 column 1 (char 0)

  at /usr/local/lib/python3.10/json/decoder.py:355 in raw_decode
      351│         """
      352│         try:
      353│             obj, end = self.scan_once(s, idx)
      354│         except StopIteration as err:
    → 355│             raise JSONDecodeError("Expecting value", s, err.value) from None
      356│         return obj, end
      357│ 
```

### Deployment

Docker (Linux)

### Deployment Details

I'm on Debian 11 (bullseye)

**Comment by Miroito** [comment]
I ran into this issue as well, #1676 fixes it by using an upgraded version of poetry.

**Comment by hay-kot** [comment]
Fixed in #1694
